Offseason2022/src/main/java/frc/robot/commands/ScoringActionHighHub.java

```diff
@@ -3,27 +3,42 @@
 
 package frc.robot.commands;
 
+import edu.wpi.first.wpilibj2.command.ParallelDeadlineGroup;
 import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import edu.wpi.first.wpilibj2.command.WaitUntilCommand;
+import frc.robot.Constants.FCConsts.FCMode;
+import frc.robot.Constants.INConsts.INMode;
+import frc.robot.Constants.SHConsts.Mode;
+import frc.robot.Constants.TCConsts.TCMode;
+import frc.robot.subsystems.FloorConveyor;
+import frc.robot.subsystems.Intake;
 import frc.robot.subsystems.Shooter;
+import frc.robot.subsystems.TowerConveyor;
 
 /**
  *
  */
 public class ScoringActionHighHub extends SequentialCommandGroup
 {
-  public ScoringActionHighHub(double waitTime, Shooter shooter)
+  public ScoringActionHighHub(double waitTime, Intake intake, FloorConveyor fConv, TowerConveyor tConv, Shooter shooter)
   {
+    setName("ScoringActionHighHub");
+
     addCommands(
-    // Add Commands here:
-    // Also add parallel commands using the
-    //
-    // addCommands(
-    // new command1(argsN, subsystem),
-    // parallel(
-    // new command2(argsN, subsystem),
-    // new command3(argsN, subsystem)
-    // )
-    // );
+        // Add Commands here:
+
+        // @formatter:off
+      new ParallelDeadlineGroup(
+        new WaitUntilCommand(shooter::isAtDesiredSpeed), 
+        new ShooterRun(shooter, Mode.SHOOTER_UPPERHUB)),

```

Comment:
Let's use the style you have below. Swing the second parens on line 34 around and put it on its own line. Everything else looks good. You'll see the same issue in both LowHub and HighHub.

Reply:
Except I see both using HighHub while one should be using constants and name for LowHub.